Lib/test/test_ipaddress.py

```diff
@@ -2548,6 +2548,24 @@ def testsixtofour(self):
                          sixtofouraddr.sixtofour)
         self.assertFalse(bad_addr.sixtofour)
 
+    # issue41004 Hash collisions in IPv4Interface and IPv6Interface
+
+    def testV4HashIsNotConstant(self):
+        ipv4_address1 = ipaddress.IPv4Interface("1.2.3.4")
+        ipv4_address2 = ipaddress.IPv4Interface("2.3.4.5")
+        self.assertNotEqual(32, ipv4_address1.__hash__())
```

Comment:
Please add a comment here about why the number 32 is being tested.

Reply:
@ericvsmith 
I was wondering if this assertion makes any sense at all now and that asserting that the hashes of two different objects is not equal is enough.

What do you think?

Reply:
I could go either way, but I'd probably drop it if I were writing it myself. In any event, I think it needs a comment about why the test exists. Just mentioning the bpo number is good enough ("See bpo-41004").

Reply:
I am going the lose the assertion.